js/gated-content/src/components/event/EventTeaser.vue

```diff
@@ -68,22 +66,28 @@ export default {
   },
   computed: {
     date() {
-      return moment(this.video.attributes.date.value).format('YYYY-MM-DD');
+      return dayjs(new Date(this.video.attributes.date.value)).format('YYYY-MM-DD');
```

Comment:
@retif , according to this documentation, dayjs creates a wrapper for the Date object inside itself.
https://day.js.org/docs/en/parse/parse
It looks like that we can use:
```suggestion
      return dayjs(this.video.attributes.date.value).format('YYYY-MM-DD');
```
Instead of your code.